src/main/webapp/app/tasks/task-form/task-form.component.html

```diff
@@ -1,6 +1,12 @@
 <form [formGroup]="taskForm" (ngSubmit)="onSubmit()">
   <mat-form-field>
     <input type="text" formControlName="name" matInput placeholder="Add a new tiny task…" data-cy="task-input">
-    <mat-icon matSuffix aria-label="Add task"(click)="onSubmit()">send</mat-icon>
   </mat-form-field>
+  <mat-form-field appearance="fill">
```

Comment:
this appearances gives it some nasty layout and background shade, would have gone without it